tests/Functional/DiffTest.php

```diff
@@ -69,6 +69,6 @@ public function testExecute()
 
     public static function tearDownAfterClass()
     {
-        (new Filesystem())->remove(self::$cwd);
+        (new Filesystem())->remove(sys_get_temp_dir().'/Manala');
```

Comment:
Hummm...okay, why not. But I think its preferable to keep the code as it was before and only removing the directory created in the `setUpBeforeClass`.